client/app/cloud/offer/cloud-offer.controller.js

```diff
@@ -1,15 +1,15 @@
 (() => {
   class CloudOfferCtrl {
     constructor($q, $stateParams, $translate, CucFeatureAvailabilityService, CloudProjectAdd,
-      CucCloudMessage, OvhApiMe, TARGET, URLS) {
+      CucCloudMessage, OvhApiMe, coreConfig, URLS) {
       this.$q = $q;
       this.$stateParams = $stateParams;
       this.$translate = $translate;
       this.CloudProjectAdd = CloudProjectAdd;
       this.CucCloudMessage = CucCloudMessage;
       this.User = OvhApiMe;
       this.CucFeatureAvailabilityService = CucFeatureAvailabilityService;
-      this.TARGET = TARGET;
+      this.region = coreConfig.getRegion();
```

Comment:
I'm just wondering why we didn't use getters/setters inside the class definition. Is there any reason?

Reply:
I replaced the minimum of things to avoid a big pull request, but yes, we can use getters inside class methods :)